scipy/stats/_distribution_infrastructure.py

```diff
@@ -3651,6 +3664,47 @@ class or its methods for more information.
     >>> np.isclose(X.cdf(2.1), Y.cdf(2.1))
     np.True_
 
+    Create a custom distribution with multiple parameterizations. Here we create a
+    custom version of the beta distribution that has an alternative parameterization
+    in terms of the mean ``mu`` and a dispersion parameter ``nu``.
+
+    >>> class MyBeta:
+    ...     @property
+    ...     def __make_distribution_version__(self):
+    ...         return "1.16.0"
+    ...
+    ...     @property
+    ...     def parameters(self):
+    ...         return (
+    ...             {"a": (0, np.inf), "b": (0, np.inf)},
+    ...             {"mu": (0, 1), "nu": (0, np.inf)},
+    ...         )
+    ...
+    ...     def process_parameters(self, a=None, b=None, mu=None, nu=None):
+    ...         if a is not None and b is not None and mu is None and nu is None:
+    ...             nu = a + b
+    ...             mu = a / nu
+    ...         else:
+    ...             a = mu * nu
+    ...             b = nu - a
+    ...         return {"a": a, "b": b, "mu": mu, "nu": nu}
```

Comment:
While I'm changing things completely subjectively...
```suggestion
    ...         return dict(a=a, b=b, mu=mu, nu=nu)
```